[Chore] v13.0-rc1 release (#445)
serokell/tezos-packaging: docs/baking.md

### Installation

In order to run a baking instance, you'll need the following Tezos binaries:
`tezos-client`, `tezos-node`, `tezos-baker-<proto>`.

> The currently supported protocol is `012-Psithaca` (used on `ithacanet` and `mainnet`) and

Reviewer: A similar line needs to be updated in the setup wizard too.

> `013-PtJakart` (jakartanet).

Reviewer 2: ```suggestion
`013-PtJakart` (`jakartanet`).
```

In order to run a baking instance, you'll need the following Tezos binaries:
`tezos-client`, `tezos-node`, `tezos-baker-<proto>`, `tezos-endorser-<proto>`.

> The currently supported protocol is `012-Psithaca` (used on `ithacanet` and `mainnet`) and
> `013-PtJakart` (jakartanet).

Reviewer: ```suggestion
The currently supported protocols are `012-Psithaca` (used on `ithacanet` and `mainnet`) and
`013-PtJakart` (used on `jakartanet`).
```

Reviewer: @rvem I un-resolved this because it still applies after some fixups (there is a `ised` typo and it's not using the plural)
